curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/adjust-the-background-color-property-of-text.chinese.md

```diff
@@ -2,27 +2,36 @@
 id: 587d781b367417b2b2512abc
 title: Adjust the background-color Property of Text
 challengeType: 0
-videoUrl: ''
-localeTitle: 调整文本的背景颜色属性
+videoUrl: 'https://scrimba.com/c/cEDqwA6'
+forumTopicId: 301032
+localTitle: 调整文本的背景色
```

Comment:
I'm not sure if it's should be `localeTitle`?

Reply:
Yes, it's a mistake, I will fix this.